tests/util/kartutil.go

```diff
@@ -295,10 +300,41 @@ func K8sExecInPodWithContainer(pod string, ns string, container string, cmd []st
 	return buf.String(), errBuf.String(), nil
 }
 
+func isBpfLsmEnforcer() (bool, error) {
+	if _, err := os.Stat("/tmp/karmorProbeData.cfg"); err != nil {
+		log.Errorf("unable to karmorProbeData.cfg: %s", err)
+		return false, nil
```

Comment:
We don't use host process mode but deploy kubearmor on k8s in tests. Not sure if this would work. We should add the check to extract as well maybe?